### PR TITLE
Add e2e test doc

### PR DESCRIPTION
**Pull request by QuakeWang**

### CODE REVIEW

**Comment by QuakeWang** [comment]
@zhongjiajie PTAL

**Comment by zhongjiajie** [comment]
@QuakeWang  Do we have English version description?

**Comment by QuakeWang** [comment]
> @QuakeWang Do we have English version description?

If there are no problems with the Chinese version, I will translate it later.

**Comment by zhongjiajie** [comment]
I would take a quick review, PLS wait a minus

File: development/zh-cn/e2e-test.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+Selenium 框架通过一个面向用户的界面将所有这些部分连接在一起， 该界面允许透明地使用不同的浏览器后端， 从而实现跨浏览器和跨平台自动化。
+
+## 二、E2E 测试
+
+### 1、E2E-Pages
+
+本次测试使用 docker-compose 部署，当前测试的为单机模式，主要用于检验一些例如“增删改查”基本功能，后期如需做集群验证，例如不同服务之间的协作，或者各个服务之间的通讯机制，可参考 `deploy/docker/docker-compose.yml`来配置。
+
+对于 E2E 测试（前端这一块），使用页面模型的形式，主要为每一个页面建立一个对应的模型。下面以登录页为例：
```
Comment (zhongjiajie) [inline development/zh-cn/e2e-test.md:29]:
```suggestion
对于 E2E 测试（前端这一块），使用 [页面模型](https://www.selenium.dev/documentation/guidelines/page_object_models/) 的形式，主要为每一个页面建立一个对应的模型。下面以登录页为例：
```

File: development/zh-cn/e2e-test.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+        page.create(tenant);
+
+        await().untilAsserted(() -> assertThat(page.tenantList())
+                .as("Tenant list should contain newly-created tenant")
+                .extracting(WebElement::getText)
+                .anyMatch(it -> it.contains(tenant)));
+    }
+```
+
+其余的都是类似的情况，可参考具体的源码来理解。
```
Comment (zhongjiajie) [inline development/zh-cn/e2e-test.md:177]: Should we copy-paste the link of E2E test case https://github.com/apache/dolphinscheduler/tree/dev/dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/java/org/apache/dolphinscheduler/e2e/cases ?

**Review by zhongjiajie — COMMENTED** [review]
And NIT, others LGTM, Thanks @QuakeWang

File: development/zh-cn/e2e-test.md (hunk 2 above)
Comment (QuakeWang) [reply development/zh-cn/e2e-test.md:177]: Good idea!

**Comment by QuakeWang** [comment]
> And NIT, others LGTM, Thanks @QuakeWang

I have updated it : )

**Comment by zhongjiajie** [comment]
> > @QuakeWang Do we have English version description?
> 
> If there are no problems with the Chinese version, I will translate it later.

Now, LGTM, maybe it is time to translate to English now

**Comment by QuakeWang** [comment]
> > > @QuakeWang Do we have English version description?
> > 
> > 
> > If there are no problems with the Chinese version, I will translate it later.
> 
> Now, LGTM, maybe it is time to translate to English now

I have submitted the English version, please check it out.

**Review by zhongjiajie — APPROVED** [review]
LGTM, thanks @QuakeWang

**Pull request merged**